app/views/admin/cache_settings/edit.html.haml

```diff
@@ -1,6 +1,19 @@
 - content_for :page_title do
   = t(:cache_settings)
 
+= form_tag main_app.admin_cache_settings_path, :method => :put do
+  .field
+    = hidden_field_tag 'preferences[enable_products_cache?]', '0'
+    = check_box_tag 'preferences[enable_products_cache?]', '1',  Spree::Config[:enable_products_cache?]
+    = label_tag nil, t('.enable_products_cache')
+  .form-buttons
+    = button t(:update), 'icon-refresh'
+
+%br
+%br
+
+%h4= t(:cache_state)
+%br
```

Comment:
are we sure that allowing any superadmin to play with this is the right move? IMO this belongs to code-level config only available to be modified by devs, which is something different than settings.

Besides, settings page already looks like a hard to manage Frankenstein.

I know we've added settings for other similar things and it seems unfair but IMO this distinction is pretty important. What do you guys think?

Reply:
I just dont want to have to login to the server to debug this in the process of debugging v2. So, we need this now in ofn v2, that's the reason I built it, and that's why it's already deployed in fr staging :-D
I think this is a lot more useful if available to superadmins.
Also, this is not a big risk at all, the cache will keep being updated, it will just not be picked up by the shopfront renderer.
So, I vote for inclusion of this option in the existing config page.

I think we can plan to split the config pages into two different groups: the app-admin pages and the system-admin pages like this one.